lib/unimatrix/regent_sdk/email_settings_request.rb

```diff
@@ -15,16 +10,23 @@ def initialize( realm_uuid, access_token )
       end
       
       def retrieve
-        uri = URI( REGENT_URL + '/realms/' + @realm_uuid + '?access_token=' + @access_token + '&include[settings]' )
-        response = Net::HTTP.get( uri )
-        all_realm_settings = JSON.parse( response )[ 'settings' ]
-        email_settings = all_realm_settings.select { | setting | setting[ 'name' ][ 0..5 ] == 'email_' }
-        email_settings.map do | setting | 
-          setting.keep_if { | key, value | [ 'name', 'content' ].include?( key ) } 
-          setting[ 'name' ] = setting[ 'name' ][ 6..-1 ]
-        end
+        request = Request.new( 
+                    '/realms/' + @realm_uuid, 
+                    { access_token: @access_token, include_settings: true } 
+                  )
+                  
+        response = request.get
         
-        email_settings
+        if response.success?
```

Comment:
- You should make the class more generic `Unimatrix::Regent::Settings` instead of `Unimatrix::Regent::EmailSettingsRequest`. The reason is that later on, you may want to allow requests for settings other than email. Use a setting_type param to filter for settings. E.g.
```
Unimatrix::Regent::Settings.new().query( setting_type: 'email' )
```
- To make sure that the user can't request a private settings, you can have a list of allowed settings and check it agains the setting_type:
```
ALLOWED_SETTINGS = ['email', 'artifact_type', ...]
```

I assume KCR approved the name of the SDK? I think Unimatrix::Regent is sufficient/more concise than Unimatrix::RegentSdk (also more consistent with Unimatrix::Authorization, etc).